### DISCUSSION TITLE
Is it dead?

### DISCUSSION THREAD

**Discussion by sooLz28** [Ludeon/RimWorld-ar#15] [General]
Is the project dead?

**Comment by mtimoustafa** [comment]
@sooLz28 👋🏽  There hasn't been any activity in the project, but you're welcome to pick up the work yourself and get other people to contribute too.

I have no time to contribute to it currently, but take a look at https://github.com/mtimoustafa/rimworld-rtl-translation-tools if you want a way to format the strings properly so it works with the game.

**Reply by mtimoustafa** [reply]
I also see @abduzy2003 has been committing some changes recently. We also have a PR open from @zxsoszx.

Considering the game is still actively developed, it's a good chance to get @Ludeon/translators-arabic active again 😄

**Reply by CGUltimateno** [reply]
I'm all up to getting it started and ready to assist with it. We should make like a Discord server to start getting in contact more.

**Reply by mtimoustafa** [reply]
That's a good idea @CGUltimateno. If you make one feel free to invite me, I'm `@fairewinds` on Discord 🙂 

Also, I'm looking at a tool called [RimTranslate](https://github.com/winterheart/RimTranslate), which might make the translation process easier. It's a bit outdated, so I forked it to https://github.com/mtimoustafa/RimTranslate and will start polishing it.

We can also integrate https://github.com/mtimoustafa/rimworld-rtl-translation-tools into it so we have a simple way of generating translation files, that can work as a single source of truth.

**Reply by mtimoustafa** [reply]
@sooLz28 you can also look into joining @Ludeon/translators-arabic. Someone from @Ludeon should be able to add you if you're still interested.

**Reply by sooLz28** [reply]
So, when would they add the translation to the game? when it's 100% completed?

**Reply by sooLz28** [reply]
@mtimoustafa  I have been into https://github.com/mtimoustafa/rimworld-rtl-translation-tools and its very nice!

**Reply by mtimoustafa** [reply]
> So, when would they add the translation to the game? when it's 100% completed?

Good question - I don't want to assume here so we should also ask @Ludeon.
